include/nbl/builtin/hlsl/cpp_compat/output_structs.hlsl

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2023 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+#ifndef _NBL_BUILTIN_HLSL_SPIRV_INTRINSICS_OUTPUT_STRUCTS_INCLUDED_
+#define _NBL_BUILTIN_HLSL_SPIRV_INTRINSICS_OUTPUT_STRUCTS_INCLUDED_
+
+#include <nbl/builtin/hlsl/concepts/core.hlsl>
+#include <nbl/builtin/hlsl/concepts/vector.hlsl>
+#include <nbl/builtin/hlsl/vector_utils/vector_traits.hlsl>
```

Comment:
maybe `tgmath/output_structs.hlsl`? because std::modf is in cmath which we decided to have in tgmath?

https://en.cppreference.com/w/cpp/numeric/math/modf